var/spack/repos/builtin/packages/curl/package.py

```diff
@@ -8,6 +8,8 @@
 import re
 import sys
 
+from llnl.util.filesystem import compute_sfn
```

Comment:
(for later PR) I'm inclined to say this should be made available by `build_environment.set_package_py_globals`

Reply:
I think CURL is a bit unique in how bad it is an handling paths with spaces simply due to the volume of code generation involved, but for a later PR I'll look at some of the other NMake packages and see if this is worthwhile.

Reply:
Actually I was thinking of perl, this should absolutely be in `build_environment`